config/initializers/hesa_degree_types.rb

```diff
@@ -82,5 +82,4 @@
   [400, nil, 'First Degree', :other],
   [401, nil, 'Higher Degree', :other],
   [402, nil, 'Degree equivalent', :other],
-  [999, nil, 'Unknown', :other],
```

Comment:
Thoughts on this removal? It's the simplest way to remove if from the frontend and it doesn't seem to serve any purpose for this feature.